ldap/docker-root/georchestra.ldif

```diff
@@ -96,7 +96,7 @@ sn: geoserver_privileged_user
 mail: psc+geoserver_privileged_user@georchestra.org
 uid: geoserver_privileged_user
 cn: geoserver_privileged_user
-description: Do not modify.  This is a required user for extractorapp, mapfishapp...
+description: Do not modify.  This is a required user for extractorapp, ...
```

Comment:
same comment, maybe we shall rephrase it like "this is a required user for server-to-server internal communications"